File: redsheet-portal/src/app/database-models/v-language-content-name.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLanguageContentName {
    constructor(vLanguageContentName?: vLanguageContentName )  {
        let properties = [ "english", "languageContentId", "languageContentName",];
        for (let property of properties)
            if (vLanguageContentName && vLanguageContentName[property])
                this[property] = vLanguageContentName[property];
    }
 
	english : string =   undefined;
 
	languageContentId : number =   0 ;
 
    @required()
    @maxLength(50)
	languageContentName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vLanguageContentName.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLanguageContentNames", Schema = "dbo")]
	[Serializable]
	public partial class vLanguageContentName  
	{

	public string English { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LanguageContentId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LanguageContentName { get; set; } 

	public vLanguageContentName()
	{
    }

    }
}

